Ownership/TC - LTC/London TC COWLY.txt

```diff
@@ -31,22 +31,22 @@ GUEST:L:*:EGBB
 GUEST:L:*:EGBE
 
 SECTOR:London TC COWLY 3 (DB-225):0:22500
-OWNER:COWLY:TCM:LCW:LC:LNC:LWC:LSC:L
-ALTOWNER:Observing LON_C_CTR:TCGS:COWLY:TCM:LCW:LGS:O:LM:LC:LNC:LWC:LSC:L
-ALTOWNER:Observing LON_SC_CTR:TCGS:COWLY:TCM:LCW:LC:LNC:LWC:LGS:O:LM:LSC:L
-ALTOWNER:Observing London FIR:TCGS:LGS:O:LM:COWLY:TCM:LCW:LC:LNC:LWC:LSC:L
-BORDER:LC6:LC4:LC7a:LC7b:LC8:LWC4b
+OWNER:COWLY:TCM:LCW:LC:LNC:LSC:L
+ALTOWNER:Observing LON_C_CTR:TCGS:COWLY:TCM:LCW:LGS:O:LM:LC:LNC:LSC:L
+ALTOWNER:Observing LON_SC_CTR:TCGS:COWLY:TCM:LCW:LC:LNC:LGS:O:LM:LSC:L
+ALTOWNER:Observing London FIR:TCGS:LGS:O:LM:COWLY:TCM:LCW:LC:LNC:LSC:L
+BORDER:LC6:LC4:LC7a:LC7b:LC8
 
 SECTOR:London TC COWLY 4 (DB-215):0:21500
-OWNER:COWLY:TCM:LCW:LC:LNC:LWC:LSC:L
-ALTOWNER:Observing LON_C_CTR:TCGS:COWLY:TCM:LCW:LGS:O:LM:LC:LNC:LWC:LSC:L
-ALTOWNER:Observing LON_SC_CTR:TCGS:COWLY:TCM:LCW:LC:LNC:LWC:LGS:O:LM:LSC:L
-ALTOWNER:Observing London FIR:TCGS:LGS:O:LM:COWLY:TCM:LCW:LC:LNC:LWC:LSC:L
-BORDER:LC7a:LC7b:LC8:LWC4c:LWC4d:LTC22c:LTC1a:LTC1b:LC19
+OWNER:COWLY:TCM:LCW:LC:LNC:LSC:L
+ALTOWNER:Observing LON_C_CTR:TCGS:COWLY:TCM:LCW:LGS:O:LM:LC:LNC:LSC:L
+ALTOWNER:Observing LON_SC_CTR:TCGS:COWLY:TCM:LCW:LC:LNC:LGS:O:LM:LSC:L
+ALTOWNER:Observing London FIR:TCGS:LGS:O:LM:COWLY:TCM:LCW:LC:LNC:LSC:L
+BORDER:LC7a:LC7b:LC8:LTC22c:LTC1a:LTC1b:LC19
 
 SECTOR:London TC COWLY 5 (165-215):16500:21500
-OWNER:COWLY:TCM:LCW:LC:LNC:LWC:LSC:L
-ALTOWNER:Observing LON_C_CTR:TCGS:COWLY:TCM:LCW:LGS:O:LM:LC:LNC:LWC:LSC:L
-ALTOWNER:Observing LON_SC_CTR:TCGS:COWLY:TCM:LCW:LC:LNC:LWC:LGS:O:LM:LSC:L
-ALTOWNER:Observing London FIR:TCGS:LGS:O:LM:COWLY:TCM:LCW:LC:LNC:LWC:LSC:L
```

Comment:
Line erroneously amended

Reply:
What is wrong in regards to this one? TC COWLY 5